File: python/raindrops/raindrops.py
def convert(number):
    r3 = "Pling" if number % 3 == 0 else ""
    r5 = "Plang" if number % 5 == 0 else ""
    r7 = "Plong" if number % 7 == 0 else ""
    mg = "%s%s%s" % (r3, r5, r7)
    return mg if mg else str(number)

File: clojure/raindrops/src/raindrops.clj
(ns raindrops)

(defn drops [n]
   (letfn [(fac [x] (zero? (rem n x)))]
      (str 
         (if (fac 3) "Pling" "")
         (if (fac 5) "Plang" "")
         (if (fac 7) "Plong" ""))))

(defn convert [n] 
   (let [msg (drops n)]
      (if (empty? msg) (str n) msg)))
